src/ui/foundation/GlobalStateProvider/context.ts

```diff
@@ -1,15 +1,9 @@
 import { createContext } from 'react';
-import { UiMain, uiMainInstance } from '@/ui/uiMain';
+import { UiMain } from '@/ui/uiMain';
 
 export interface GlobalState {
   activeVaultId: string;
   uiMain: UiMain;
 }
 
-export const defaultGlobalState: GlobalState = {
-  activeVaultId: '',
-  uiMain: uiMainInstance,
-};
-
-export const GlobalStateContext =
-  createContext<GlobalState>(defaultGlobalState);
+export const GlobalStateContext = createContext<GlobalState>({} as GlobalState);
```

Comment:
This here is the equivalent of refusing to provide a default value.

This should be perfectly fine:
- The default value is only used when calling useContext outside the provider.
- This should crash anyways.
- This will make it crash.
- Everyone is happy?

Stackoverflow told me to to this:
- https://stackoverflow.com/a/61336826